android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -76,8 +76,8 @@
 		"adjective": ["Chinese"],
 		"preferredVictoryType": "Scientific",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Heaven itself bends before your beauty, Empress Wu Zetian, ruler of eternal China! You lead one of the oldest and greatest civilization of humanity. In the old ages, chinese people already mastered a level of culture and science that many other civilizations will reach much later in history. China brought so many contributions to human knowledge - the printing press, gunpowder, the works of Confucius -, and this, long before the rest of the world was mature enough to make the same discoveries!",
```

Comment:
Point here is repeated. Instead:

"In your time, China stood at the forefront of science and technology, bringing the world such contributions as gunpowder, the printing press, and the world of Confucius."